src/commissioner/commissioner.cpp

```diff
@@ -354,13 +355,14 @@ void Commissioner::HandleCommissionerSet(const Coap::Message &aMessage, void *aC
             switch (state)
             {
             case Meshcop::kStateAccepted:
-                commissioner->mCommissionState = kStateAccepted;
+                commissioner->mCommissionState = CommissionState::kStateAccepted;
```

Comment:
I don't think we should set `mCommissionState` here. This only rejects or accepts the set data.